### DISCUSSION TITLE
Highlighting Searched Text

### DISCUSSION THREAD

**Discussion by bnpwalter** [freeplane/freeplane#845] [Ideas]
LibreOffice has a nice feature that allows you to highlight text color for searched text.  The existing FP search function is great however sometimes in preparing large text documents it would be helpful if the searched text would highlight so that one did not have to search within the node activated by the search function.

**Comment by euu2021** [comment]
Is the request to have the highlighting directly inside the node?

Maybe, while your request is not fulfilled, the [Jumper add-on](https://github.com/lilive/Freeplane-Jumper) could improve things a little bit for you, as it has highlighting in the search results (but, not directly in the node).

**Comment by bnpwalter** [comment]
Yes. I was thinking more about highlighting within nodes.  Will look at the jumper add-on.  Thanks.
